ocs_ci/utility/utils.py

```diff
@@ -5198,3 +5199,38 @@ def extract_image_urls(string_data):
     # Find all URLs that start with 'registry.redhat.io'
     image_urls = re.findall(r'registry\.redhat\.io[^\s"]+', string_data)
     return image_urls
+
+
+def params_to_configini_file(params):
```

Comment:
rename the function which resembles the  work of function. In this case it creates  config ini file. ( e.g: create_config_ini_file()